### PR TITLE
fix(dotnet): remove create readme logic in solution for dotnet project

### PR DESCRIPTION
**Pull request by JerryYangKai**
E2E TEST: no need.

### CODE REVIEW

File: packages/fx-core/src/plugins/solution/fx-solution/v2/utils.ts [hunk 1]
```diff
@@ -54,6 +54,10 @@ export function isAzureProject(azureSettings: AzureSolutionSettings | undefined)
   return azureSettings !== undefined && HostTypeOptionAzure.id === azureSettings.hostType;
 }
 
+export function isVsPlatform(ctx: v2.Context): boolean {
```
Comment (eriolchan) [inline packages/fx-core/src/plugins/solution/fx-solution/v2/utils.ts:57]: @hund030  Just curious if there is any common place to put this isVsPlatform() logic? I am not sure if this method exist in both solution, tab and bot plugin?

Comment (hund030) [reply packages/fx-core/src/plugins/solution/fx-solution/v2/utils.ts:57]: Yes. Maybe we can put this in common/tools.ts.

Comment (JerryYangKai) [reply packages/fx-core/src/plugins/solution/fx-solution/v2/utils.ts:57]: Good idea！

Comment (jayzhang) [reply packages/fx-core/src/plugins/solution/fx-solution/v2/utils.ts:57]: we already have this function globally: 
image

**Comment by kuojianlu** [comment]
Any effect on opening readme behavior after project creation? https://github.com/OfficeDev/TeamsFx/blob/dev/packages/fx-core/src/core/index.ts#L231

File: packages/fx-core/src/plugins/solution/fx-solution/v2/utils.ts (hunk 1 above)
Comment (eriolchan) [reply packages/fx-core/src/plugins/solution/fx-solution/v2/utils.ts:57]: Thanks for sharing @jayzhang, @JerryYangKai can you help check if we can leverage this?

**Comment by JerryYangKai** [comment]
> Any effect on opening readme behavior after project creation? https://github.com/OfficeDev/TeamsFx/blob/dev/packages/fx-core/src/core/index.ts#L231

This is only triggered when this project is created from VS, so I think no effect to VS code part. Any suggestions?

**Comment by kuojianlu** [comment]
> > Any effect on opening readme behavior after project creation? https://github.com/OfficeDev/TeamsFx/blob/dev/packages/fx-core/src/core/index.ts#L231
> 
> This is only triggered when this project is created from VS, so I think no effect to VS code part. Any suggestions?

I think for VS, there is no need to scaffold readme, thus no need to set `openReadme` and `automaticNpmInstall` state in https://github.com/OfficeDev/TeamsFx/blob/dev/packages/fx-core/src/core/index.ts#L231. @jayzhang What do you think?

**Review by jayzhang — APPROVED** [review]

**Pull request merged**